flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/table/FileSystemCommitterTest.java

```diff
@@ -198,6 +240,81 @@ void testEmptyPartition() throws Exception {
         assertThat(new File(emptyPartitionFile, "f1")).exists();
     }
 
+    @Test
+    public void testWriteSuccessFile() throws Exception {
```

Comment:
Do we need this test. Can we check successfile in method `testEmptyPartition`, `testPartition`, `testNotPartition` ?

Reply:
good catch